server/graphql/types/src/types/vaccination.rs

```diff
@@ -13,7 +18,49 @@ impl VaccinationNode {
         &self.row().id
     }
 
-    // TODO
+    pub async fn vaccination_date(&self) -> &NaiveDate {
```

Comment:
Sorry @CarlosNZ, depending on what you've done for the card we may have doubled up a little here. Don't think it will be too much to resolve the conflicts though 🤞